data/battle_anim_scripts.s

```diff
@@ -729,7 +742,29 @@ gBattleAnims_Moves::
 	.4byte Move_METEOR_ASSAULT
 	.4byte Move_ETERNA_BEAM
 	.4byte Move_STEEL_BEAM
-	.4byte Move_COUNT @ cannot be reached, because last move is Psycho Boost
+    .4byte Move_EXPANDING_FORCE
+    .4byte Move_STEEL_ROLLER
+    .4byte Move_SCALE_SHOT
+    .4byte Move_METEOR_BEAM
+    .4byte Move_SHELL_SIDE_ARM
+    .4byte Move_MISTY_EXPLOSION
+    .4byte Move_GRASSY_GLIDE
+    .4byte Move_RISING_VOLTAGE
+    .4byte Move_TERRAIN_PULSE
+    .4byte Move_SKITTER_SMACK
+    .4byte Move_BURNING_JEALOUSY
+    .4byte Move_LASH_OUT
+    .4byte Move_POLTERGEIST
+    .4byte Move_CORROSIVE_GAS
+    .4byte Move_COACHING
+    .4byte Move_FLIP_TURN
+    .4byte Move_TRIPLE_AXEL
+    .4byte Move_DUAL_WINGBEAT
+    .4byte Move_SCORCHING_SANDS
+    .4byte Move_JUNGLE_HEALING
+    .4byte Move_WICKED_BLOW
+    .4byte Move_SURGING_STRIKES
```

Comment:
Just like above, the wrong indentation style is being used here. Should be tabs, not spaces.